source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.ApplyDBMigrationsApp/Program.cs

```diff
@@ -22,15 +24,27 @@ public static class Program
         public static int Main(string[] args)
         {
             var connectionString = ConnectionStringFactory.GetConnectionString(args);
-            var filter = EnvironmentFilter.GetFilter(args);
             var isDryRun = args.Contains("dryRun");
 
+            var preDeployResult = PerformUpgrade(connectionString, EnvironmentFilter.GetPreDeployFilter(args), isDryRun);
+            if (!preDeployResult.Successful) { return ResultReporter.ReportResult(preDeployResult); }
+            ResultReporter.ReportResult(preDeployResult);
+
+            var result = PerformUpgrade(connectionString, EnvironmentFilter.GetFilter(args), isDryRun);
+            if (!preDeployResult.Successful) { return ResultReporter.ReportResult(result); }
+            ResultReporter.ReportResult(result);
```

Comment:
Think this a copy paste thing

```suggestion
            var result = PerformUpgrade(connectionString, EnvironmentFilter.GetFilter(args), isDryRun);
            if (!result.Successful) { return ResultReporter.ReportResult(result); }
            ResultReporter.ReportResult(result);
```

Reply:
Correct, bug is fixed now.